blockchain/stake/internal/tickettreap/common.go

```diff
@@ -87,11 +88,27 @@ type Value struct {
 type treapNode struct {
 	key      Key
 	value    *Value
-	priority int
+	size     int // how many items are within this treap, including counting this node as 1
+	priority uint32 // use blockheight as priority, this makes it trivial to search by blockheight
 	left     *treapNode
 	right    *treapNode
 }
 
+// constructString is a helper function for Immutable.String()
+func (t *treapNode) constructString(indent string) string {
```

Comment:
There is no need for this function.  You can just do:

```
import "github.com/davecgh/go-spew/spew"
...
spew.Dump(sn.liveTickets)
```

Also, as a general rule, we don't want debug code in the production code.  If something like this is really desired, it should be put into the test package.  However, there is almost never a need for it since you can just use `spew` to dump data structs whenever you want.